nichtparasoup.py

```diff
@@ -345,7 +338,7 @@ def on_reset(self, request):
         cache_fill_thread.start()
     except (KeyboardInterrupt, SystemExit):
         # end the cache filler thread properly
-        min_cache_imgs = -1  # stop cache_fill-inner_loop
+        min_cache_imgs_before_refill -1;
```

Comment:
I dont think this makes any sense, i hope there is a better solution than this.
For example: Maybe a variable "runNichtparasoup" in the cache-fill-loop that gets set to false here...